cmd/evm/internal/t8ntool/transition.go

```diff
@@ -178,9 +179,20 @@ func Transition(ctx *cli.Context) error {
 	prestate.Env = *inputData.Env
 
 	vmConfig := vm.Config{
-		Tracer: tracer,
-		Debug:  (tracer != nil),
+		Tracer:           tracer,
+		Debug:            (tracer != nil),
+		EVMInterpreter:   ctx.String(utils.EVMInterpreterFlag.Name),
+		EWASMInterpreter: ctx.String(utils.EWASMInterpreterFlag.Name),
 	}
+
+	if vmConfig.EVMInterpreter != "" {
+		vm.InitEVMCEVM(vmConfig.EVMInterpreter)
+	}
+
+	if vmConfig.EWASMInterpreter != "" {
+		vm.InitEVMCEwasm(vmConfig.EWASMInterpreter)
+	}
+
```

Comment:
Support for EVMC is added to `t8ntool` (used normally by `retesteth` for filling xclient tests). So `retesteth` can fill tests with moar EVMs.